Error building Android after ejecting to bare and using `firebase-core` or `firebase-analytics`

## 🐛 Bug Report

### Environment

`SDK 37`

### Steps to Reproduce

- create new EXPO 37 project
- add `expo-firebase-core` and `expo-firebase-analytics`
- eject to bare
- `yarn android`

### Expected Behavior

App builds succesfully.

### Actual Behavior

```
FAILURE: Build failed with an exception.

* Where:
Build file '.../node_modules/expo-firebase-analytics/android/build.gradle' line: 62

* What went wrong:
A problem occurred evaluating project ':expo-firebase-analytics'.
> Could not find method platform() for arguments [com.google.firebase:firebase-bom:24.1.0] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler.
```

## Fixes

The following PR's are required to fix this issue (upgrade to Gradle 5):
- #7580 
- #7577

--- a/packages/expo-updates/CHANGELOG.md
+++ b/packages/expo-updates/CHANGELOG.md
@@ -7,3 +7,5 @@
 ### 🎉 New features
 
 ### 🐛 Bug fixes
+
+- Fixed 'unable to resolve class GradleVersion' when using Gradle 5. ([#4935](https://github.com/expo/expo/pull/7577) by [@IjzerenHein](https://github.com/IjzerenHein))

--- a/packages/expo-updates/expo-updates.gradle
+++ b/packages/expo-updates/expo-updates.gradle
@@ -1,6 +1,7 @@
 // Gradle script for downloading assets that make up an OTA update and bundling them into the APK
 
 import org.apache.tools.ant.taskdefs.condition.Os
+import org.gradle.util.GradleVersion
 
 void runBefore(String dependentTaskName, Task task) {
   Task dependentTask = tasks.findByPath(dependentTaskName);

